server/clients/manageRecallsApi/manageRecallsApiClient.ts

```diff
@@ -8,11 +9,6 @@ export interface PrisonerSearchResult {
   dateOfBirth?: string
 }
 
-export interface Recall {
-  id: string
-  nomsNumber: string
-}
-
 export interface Pdf {
```

Comment:
Can this be superseded by the `Pdf.ts` type?

Reply:
Good shouts, done
I'd left the Pdf type as thought I'd apply the types in a later PR. The Recall one was just to prove it works. But yep simple to do now